week-7/Homework/mandatory/1-debugging-practice/script.js

```diff
@@ -37,8 +37,8 @@ function submit() {
     alert("Please fill all fields!");
```

Comment:
Only one of the acceptance criteria has been missed out
> If any of the information is missing it shouldn't add the book and should show an alert
 In the if statement author needs to be checked too:
```
  if (
    title.value == null ||
    title.value == "" ||
    author.value == null ||
    author.value == "" ||
    pages.value == null ||
    pages.value == ""
  ) {
```

Reply:
I didn't see this to be honest, I thought I should only clear the console from errors